### DISCUSSION TITLE
How can I show more than 6 classes in Teaching page ?

### DISCUSSION THREAD

**Discussion by martinabarletta** [academicpages/academicpages.github.io#2418] [Q&A]
Hello,

I've built my website using the template, almost everything works fine right now.
I'm using only the Talks, Publications, Teaching and CV pages. I've created 7 teaching md files but only 6 posts are shown in the teaching page. Is there a way to show more tha 6 ? Should I reenable the Blog page and store old classes there eventually ?
Thanks !

**Comment by rjzupkoii** [comment]
Can you clarify what you mean? Do you mean that once there are more than six posts it starts to use pagination, or that after that they are just missing completely?

**Reply by martinabarletta** [reply]
There is no pagination and the post just disappear, I only have 6 items shown out of 7 md files in the teaching folders.

**Reply by rjzupkoii** [reply]
Interesting, this sounds like it might be a bug. I'll see if I can reproduce it.

**Reply by rjzupkoii** [reply]
@martinabarletta So I just tried adding more than six items and they generated just fine. However, I did notice that your `2024-java-M1.md` file contains zero lines of code, which would result in it not rendering anything.
